test.py

```diff
@@ -16,6 +16,50 @@ def test_peers(self):
         self.assertTrue(type(result['peers']) == list)
         self.assertTrue(len(result['peers']) >= 1)
         self.assertEqual(len(result['peers']), result['count'])
+        
+     def test_search(self):
+        result = self.loklak.search('doctor who')
+        self.assertTrue('error' in self.loklak.search())
+        self.assertTrue('statuses' in result)
+        self.assertTrue(type(result['statuses']) == list)
+        self.assertTrue(len(result['statuses']) >= 1)
+        self.assertEqual(len(result['statuses']), int(result['search_metadata']['count']))
+
+    def test_status(self):
+        result = self.loklak.status()
+        self.assertTrue('index_sizes' in result)
+        self.assertTrue('messages' in result['index_sizes'])
+        self.assertTrue('mps' in result['index_sizes'])
+        self.assertTrue('users' in result['index_sizes'])
+        self.assertTrue('queries' in result['index_sizes'])
+        self.assertTrue('accounts' in result['index_sizes'])
+        self.assertTrue('user' in result['index_sizes'])
+        self.assertTrue('followers' in result['index_sizes'])
+        self.assertTrue('following' in result['index_sizes'])
+        
+        self.assertTrue('client_info' in result)
+        self.assertTrue('RemoteHost' in result['client_info'])
+        self.assertTrue('IsLocalhost' in result['client_info'])
+        self.assertTrue('Accept-Language' in result['client_info'])
```

Comment:
Please remove this line, it really doesn't serve a purpose in testing this. :+1: